src/main/java/org/ibp/api/rest/dataset/DatasetResource.java

```diff
@@ -184,26 +188,29 @@
 	@PreAuthorize("hasAnyAuthority('ADMIN','BREEDING_ACTIVITIES','MANAGE_STUDIES', 'INFORMATION_MANAGEMENT', 'BROWSE_STUDIES')" + PermissionsEnum.HAS_MANAGE_STUDIES_VIEW)
 	@RequestMapping(value = "/{cropname}/programs/{programUUID}/studies/{studyId}/datasets/{datasetId}/observationUnits/table", method = RequestMethod.POST)
 	@ResponseBody
+	@ApiImplicitParams({
+		@ApiImplicitParam(name = "page", dataType = "integer", paramType = "query",
+			value = "Results page you want to retrieve (0..N)"),
+		@ApiImplicitParam(name = "size", dataType = "integer", paramType = "query",
+			value = "Number of records per page."),
+		@ApiImplicitParam(name = "sort", allowMultiple = false, dataType = "string", paramType = "query",
+			value = "Sorting criteria in the format: property,asc|desc. ")
+	})
 	public ResponseEntity<ObservationUnitTable> getObservationUnitTable(@PathVariable final String cropname, @PathVariable final String programUUID,
 		@PathVariable final Integer studyId, //
 		@PathVariable final Integer datasetId, //
-		@RequestBody final ObservationUnitsSearchDTO searchDTO) {
+		@RequestBody final ObservationUnitsSearchDTO searchDTO,
+		@ApiIgnore @PageableDefault(page = 0, size = 50) final Pageable pageable) {
```

Comment:
Why choose 50 as default size?

Reply:
This should be @PageableDefault(page = 0, size = PagedResult.DEFAULT_PAGE_SIZE).